examples/src/main/java/com/mapbox/navigation/examples/core/SimpleMapboxNavigationKt.kt

```diff
@@ -341,6 +340,7 @@ class SimpleMapboxNavigationKt : AppCompatActivity(), OnMapReadyCallback,
     private val routesReqCallback = object : RoutesRequestCallback {
         override fun onRoutesReady(routes: List<DirectionsRoute>) {
             originalRoute = routes[0]
+            navigationMapboxMap.drawRoutes(routes)
```

Comment:
`onRoutesReady` is called only when new routes are requested. Only when the user requests a new route or goes off route will this callback update the route. If we remove `navigationMapboxMap.drawRoutes(routes)` from `routesObserver` we won't be able to draw routes when a faster route is available, or the route refreshes. As mentioned in the [ticket](https://github.com/mapbox/mapbox-navigation-android/issues/2473#issuecomment-585682209) we should look at this fix.

Reply:
I didn't realize faster route was dependent on the routesChanged callback. I think the routesChanged shouldn't be fired if what really happened is an alternate route has been selected by the user as the routes are still the same but that's another discussion. I'll re-examine the solution. As I was considering the change suggested in the ticket I noticed what seemed like a simpler fix.

Reply:
While examining this I noticed the app didn't follow an alternate route when the replay engine was active. I made a couple adjustments that seem to remedy this issue. 

After further testing on both my phone and an emulator I can't reproduce the blinking described in the original bug post.

Reply:
I'm going to implement the [suggestion](https://github.com/mapbox/mapbox-navigation-android/issues/2473#issuecomment-585682209) in the ticket but I'm not sure it's exactly the right fix.

Reply:
Can we move this line back to where it was?

Reply:
It's back now.